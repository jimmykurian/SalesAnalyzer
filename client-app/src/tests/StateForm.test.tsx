import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow } from "enzyme";
import StateForm from "../feature/stateForm";

configure({ adapter: new Adapter() });
test("renders the component", () => {
  const props = {
    submissionHandler: () => null,
    selectedSubmission: Math.random().toString(36).substring(7),
  };
  const component = shallow(<StateForm {...props} />);
  expect(component).toMatchSnapshot();
});
