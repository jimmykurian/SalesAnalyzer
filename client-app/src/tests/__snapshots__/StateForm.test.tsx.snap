// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div>
  <div>
    <WithStyles(ForwardRef(Grid))
      container={true}
      justify="center"
    >
      <WithStyles(ForwardRef(Typography))
        gutterBottom={true}
        style={
          Object {
            "color": "#FFFFFF",
          }
        }
        variant="h6"
      >
        States
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Grid))>
  </div>
  <div>
    <form
      onSubmit={[Function]}
    >
      <button
        onClick={[Function]}
        type="button"
      >
        Add A State Record
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        Clear State Records
      </button>
      <input
        type="submit"
      />
    </form>
  </div>
</div>
`;
